frontend/src/models/fms_api_models.ts

```diff
@@ -78,11 +78,15 @@ export interface FMSDataset extends FMSTrackedModel {
     metric_report_url?: string          // An external url to a report containing metrics for the dataset run
 }
 
+export interface FMSReadset extends FMSTrackedModel {
+    sample_name: string                // The name of the sample that was processed to produce this readset
```

Comment:
You should still include in this model the rest of the Readset content even if you don't use it.